tox.ini

```diff
@@ -3,7 +3,7 @@ envlist = es, addons-devhub-editors, main, flake8, docs, assets
 
 [testenv]
 basepython = python2.7
-install_command = pip install --no-deps --exists-action=w --find-links https://pyrepo.addons.mozilla.org/wheelhouse/ --find-links https://pyrepo.addons.mozilla.org/ --no-index {opts} {packages}
+install_command = pip install --no-deps --exists-action=w --find-links https://pyrepo.addons.mozilla.org/wheelhouse/ --find-links https://pyrepo.addons.mozilla.org/ {opts} {packages}
```

Comment:
DO NOT MERGE (it's just to make sure the tests run correctly on travis without the pytest and pytest-django versions on pyrepo